README.md

```diff
@@ -47,44 +47,43 @@ Several Youtube video tutorials and overviews are also available:
+- A DSL provides a simple and readable format for manually specifying chemical
   reactions.
 - Catalyst `ReactionSystem`s provide a symbolic representation of reaction networks,
-  built on [ModelingToolkit.jl](https://github.com/SciML/ModelingToolkit.jl) and
-  [Symbolics.jl](https://github.com/JuliaSymbolics/Symbolics.jl).
+  built on [ModelingToolkit.jl](https://docs.sciml.ai/ModelingToolkit/stable/) and
+  [Symbolics.jl](https://docs.sciml.ai/Symbolics/stable/).
 - Non-integer (e.g. `Float64`) stoichiometric coefficients are supported for generating
   ODE models, and symbolic expressions for stoichiometric coefficients are supported for
   all system types.
-- The [Catalyst.jl API](http://docs.sciml.ai/Catalyst/stable/api/catalyst_api) provides
-  functionality for extending networks, building networks programmatically,
-  network analysis, and for composing multiple networks together.
+- The [Catalyst.jl API](@ref) provides functionality for extending networks,
+  building networks programmatically, network analysis, and for composing multiple
+  networks together.
 - `ReactionSystem`s generated by the DSL can be converted to a variety of
-  `ModelingToolkit.AbstractSystem`s, including symbolic ODE, SDE, and jump process
+  `ModelingToolkit.AbstractSystem`s, including symbolic ODE, SDE and jump process
   representations.
 - Conservation laws can be detected and applied to reduce system sizes, and generate
-  non-singular Jacobians, during conversion to ODEs, SDEs, and steady-state equations.
+  non-singular Jacobians, during conversion to ODEs, SDEs, and steady state equations.
 - By leveraging ModelingToolkit, users have a variety of options for generating
   optimized system representations to use in solvers. These include construction
   of dense or sparse Jacobians, multithreading or parallelization of generated
   derivative functions, automatic classification of reactions into optimized
-  jump types for Gillespie-type simulations, automatic construction of
+  jump types for Gillespie type simulations, automatic construction of
   dependency graphs for jump systems, and more.
 - Generated systems can be solved using any
-  [DifferentialEquations.jl](https://github.com/SciML/DifferentialEquations.jl)
+  [DifferentialEquations.jl](https://docs.sciml.ai/DiffEqDocs/stable/)
   ODE/SDE/jump solver, and can be used within `EnsembleProblem`s for carrying
   out parallelized parameter sweeps and statistical sampling. Plot recipes
   are available for visualizing the solutions.
-- [Symbolics.jl](https://github.com/JuliaSymbolics/Symbolics.jl) symbolic expressions
-  and Julia `Expr`s can be obtained for all rate laws and functions determining the
-  deterministic and stochastic terms within resulting ODE, SDE, or jump models.
-- [Latexify](https://github.com/korsbo/Latexify.jl) can be used to generate
+- Julia `Expr`s can be obtained for all rate laws and functions determining the
```

Comment:
This should probably be something like "Symbolics.jl symbolic expressions can be obtained for all...". Getting `Expr`s is actually not what we return in the API, and takes some more work to get...

Reply:
Changed to "Symbolic Julia `Expr`s (implemented through [Symbolics.jl](https://github.com/JuliaSymbolics/Symbolics.jl)) can be obtained for all rate laws and functions determining the
  deterministic and stochastic terms within resulting ODE, SDE or jump models."